app/packages/forms/controllers/form_results.coffee

```diff
@@ -25,3 +31,196 @@ Template.form_results.helpers
+                userId: _submission.userId.objectId
+                questionId: questionId
+                answer: answer
+                createdAt: _submission.createdAt
+        .fail (err) ->
+          console.log err
+  # Filter by user - get submissions of user
+  @autorun ->
+    # instance.submissions.remove {}
+    # instance.answers.remove {}
+    instance.forms.find().forEach (form) ->
+      query = new Parse.Query 'Submission'
+      query.equalTo('formId', form.objectId)
+      query.find()
+        .then (submissions) ->
+          submissions.forEach (submission) ->
+            _submission = submission.toJSON()
+            instance.submissions.upsert _submission.objectId, _submission
+            for questionId, answer of _submission.answers
+              instance.answers.insert
+                formId: _submission.formId
+                userId: _submission.userId.objectId
+                questionId: questionId
+                answer: answer
+                createdAt: _submission.createdAt
+        .fail (err) ->
+          console.log err
+
+Template.form_results.helpers
+  exportButtons: ->
+    instance = Template.instance()
+    buttons = []
+    # Iterate through forms for the current Survey
+    instance.forms.find({}, {sort: {order: -1}}).forEach (button) ->
+      csv = ""
+      rows = [ ['email', 'Timestamp'] ]
+      hasData = false
+      userIds = []
+      questionIds = []
+      # Step 1: get the list of Questions for the current form
+      filter = { questionId: { $in: button._questions or [] } }
+      instance.questions.find(filter).forEach (q, i) ->
+        rows[0].push "Question #{i+1}: #{formatQuestionType(q.type)}"
+        questionIds.push q.objectId
+      # Step 1.5: put the amount of questions into the file name
+      button.questionCount = questionIds.length
+      # Step 2: retrieve all submissions for the current form
+      instance.submissions.find(
+        { formId: button.objectId },
+        { sort: createdAt: -1 }
+      ).forEach (submission) ->
+        userIds.push submission.userId.objectId
+        hasData = true
+      # Step 3: iterate over Participants
+      instance.participants.find(objectId: $in: userIds).forEach (user, u) ->
+        # Col 1: Username
+        result = [ user.username ]
+        instance.submissions.find(
+          formId: button.objectId
+          'userId.objectId': user.objectId
+        ).forEach (submission) ->
+          # Col 2: Timestamp
+          result.push formatDate(submission.createdAt)
+          # Cols 3+: Answers
+          for a in questionIds
+            result.push formatAnswer(
+              submission.answers[a],
+              instance.questions.findOne(a).type
+            )
+        rows.push result
+      # Final step: Merge rows and columns into a CSV document
+      if hasData
+        i = 0
+        ilen = rows.length
+        while i < ilen
+          csv += rows[i++].join(',')
```

Comment:
I wrap every answer into double quotes and escape the actual double quotes inside like this: `"\\""`, so commas will not be interpreted within those strings, and therefore there won't be any issues with commas within answers.